wgs.yaml

```diff
@@ -202,38 +181,17 @@ workinggroups:
   label: area/wg-manifests
   leadership:
     chairs:
-    - github: PatrickXYS
-      name: Yao Xiao
-      company: AWS
     - github: kimwnasptd
       name: Kimonas Sotirchos
-      company: Arrikto
-    - github: yanniszark
-      name: Yannis Zarkadas
-      company: Arrikto
+      company: Canonical
```

Comment:
@kimwnasptd @StefanoFioravanzo Isn't @juliusvonkohout part of Manifests WG ?

Reply:
Yes, please add me as seen in the google ACLs.